src/PhpSpec/Loader/ResourceLoader.php

```diff
@@ -105,6 +105,7 @@ private function lineIsInsideMethod(int $line, ReflectionMethod $method): bool
     private function addErrorThrowingExampleToSuite(Resource $resource, Suite $suite, \Error $error) : void
     {
         $reflection = new ReflectionClass(ErrorSpecification::class);
+        /** @var ReflectionClass<Specification> $reflection */
```

Comment:
This PHPDoc seems superfluous, `$reflection` is already of this type (even more specific).
```suggestion
```

Reply:
I had to add it to make psalm happy IIRC.